reference/forms/types/options/error_mapping.rst.inc

```diff
@@ -34,7 +34,14 @@ Here are the rules for the left and the right side of the mapping:
   object, the property path is ``[indexName]``;
 * You can construct nested property paths by concatenating them, separating
   properties by dots. For example: ``addresses[work].matchingCityAndZipCode``;
-* The left side of the error mapping also accepts a dot ``.``, which refers
-  to the field itself. That means that any error added to the field is added
-  to the given nested field instead;
 * The right side contains simply the names of fields in the form.
+
+Additionally, you can set the left side to dot (``.``), which refers to any
+unmapped property or method needing validation to the given nested field instead
+of bubbling them to the form::
```

Comment:
Actually, I still find this sentence confusing and suggest to change it to something like this:

> By default, errors for any property that is not mapped will bubble up to the parent form. You can use the dot (`.`) on the left side to map errors of all unmapped properties to a particular field.

What do you think?